appengine/standard/firebase/firetactoe/templates/fire_index.html

```diff
@@ -0,0 +1,48 @@
+<html>
+  <head>
+    {% include "_firebase_config.html" %}
+    <link rel="stylesheet" type="text/css" href="static/main.css" />
+    <script src="static/main.js"></script>
+    <script>
+      initGame('{{ game_key }}', '{{ me }}', '{{ token }}', '{{ channel_id }}',
```

Comment:
I would prefer you to store these as [`data-`](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_data_attributes) attributes on some well-known element (`body` would be fine).